app/controllers/dradis/plugins/csv/upload_controller.rb

```diff
@@ -2,24 +2,44 @@ module Dradis::Plugins::CSV
   class UploadController < ::AuthenticatedController
     include ProjectScoped
 
-    before_action :load_attachment, only: [:new]
+    before_action :load_attachment, only: [:new, :create]
 
     def new
       @default_columns = ['Unique Identifier', 'Column Header From File', 'Type', 'Field in Dradis']
 
       @headers = ::CSV.open(@attachment.fullpath, &:readline)
+
+      @last_job = Log.new.uid
```

Comment:
See: https://github.com/dradis/dradis-ce/blob/develop/app/controllers/upload_controller.rb#L20

Reply:
This will fail for concurrent calls when a different user starts a different background job. Is this different from `params[:job_id]`?

Reply:
Yes, it's different. It's how `/uploads` work console work. See: https://github.com/dradis/dradis-csv-importer/pull/2#discussion_r925309351

Reply:
This makes more sense after re-reading. `@last_job` refers to a new job and not the last job that was completed. If so, can we use a different name instead of following the upload_controller naming?

Reply:
Gotcha, i've updated it to `last_log` instead.

Reply:
Don't use `last`. What about `new_log_id` or `log_id` or `new_log`